api/features/tasks.py

```diff
@@ -50,6 +52,49 @@ def trigger_feature_state_change_webhooks(
         )
     )
 
+    if hasattr(instance.environment.project.organisation, "github_config"):
+        github_configuration = instance.environment.project.organisation.github_config
+
+        feature_data = {
+            "id": history_instance.feature.id,
+            "name": history_instance.feature.name,
+            "feature_states": [],
+        }
+        feature_data["installation_id"] = github_configuration.installation_id
+        feature_data["organisation_id"] = github_configuration.organisation.id
+
+        feature_state = {
+            "environment_name": new_state["environment"]["name"],
+            "feature_value": new_state["enabled"],
+        }
+
```

Comment:
Looks like we have the same login in models.py as well, we should make this more DRY

Reply:
Move logic to GitHub module